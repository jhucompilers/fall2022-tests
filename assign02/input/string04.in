var s;
s = "hello, world";
var n;
n = strlen(s);
println(n);
