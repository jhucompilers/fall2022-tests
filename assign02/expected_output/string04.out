12
Result: 0
